### PR TITLE
Describe major contribution aspects

### PR DESCRIPTION
**Pull request by extsoft**

### CODE REVIEW

**Comment by extsoft** [comment]
@alexbeatnik could you please review this pull request and leave your comments?

**Comment by alexbeatnik** [comment]
@extsoft Why do you use 3 commits in one pull request?

**Comment by alexbeatnik** [comment]
@extsoft  The meaning of this commit https://github.com/bees-hive/elegant-git/pull/176/commits/4ad1d287bc89ea5ca2eece20640354acb3e309b3 is understandable only after you read the file

**Comment by extsoft** [comment]
@alexbeatnik as for me, it’s not a problem having several commits in one PR. This is because they related to a single goal - describe the contribution process.

Sincerely,
Dmytro 

Sent from my iPhone

1 вер. 2019 р. о 21:49 alexbeatnik <notifications@github.com> пише:

**Comment by alexbeatnik** [comment]
> @alexbeatnik as for me, it’s not a problem having several commits in one PR. This is because they related to a single goal - describe the contribution process. Sincerely, Dmytro
> […](#)
> Sent from my iPhone 1 вер. 2019 р. о 21:49 alexbeatnik <notifications@github.com> пише:
> @extsoft Why do you use 3 commits in one pull request? — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or mute the thread.

understood thanks

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+Once the pull request is created, please make sure [status checks][status-checks] are passing.
+
+> What if the status checks are failing? If a status check is failing, and you believe that the
+failure is unrelated to your change, please leave a comment on the pull request explaining why you
+believe the failure is unrelated. A maintainer will re-run the status check for you. If we conclude
+that the failure was a false positive, then we will open an issue to track that problem with our
+status check suite.
+
+:joy::joy::joy: Thank you for the contribution! :tada::tada::tada:
```
Comment (alexbeatnik) [inline CONTRIBUTING.md]: :joy::joy::joy: i think its not realy good idea to use this smile

**Comment by extsoft** [comment]
@alexbeatnik I've fixed all your comments. Please look at changes again.

**Comment by alexbeatnik** [comment]
Thanks, looks good

**Pull request merged**